client/my-sites/stats/activity-log/index.jsx

```diff
@@ -263,7 +263,10 @@ class ActivityLog extends Component {
 
 		// loop backwards through each day in the month
 		for (
-			const m = this.applySiteOffset( moment.utc( startDate ) ).endOf( 'month' ).startOf( 'day' ),
+			const m = earliestMoment(
```

Comment:
`math.min()` would work here I think in place of the custom `earliestMoment()`?

Reply:
Turns out `moment` has `moment.min` with this exact call signature which I just discovered. Going to drop all of the redundant code and use 😅

Reply:
:D

Reply:
Cleaned up in e6b3dc8